docs/learn/avalanche/subnets-overview.md

```diff
@@ -2,107 +2,92 @@
 
 A **Subnet** is a sovereign network which defines its own rules regarding its
 membership and token economics. It is composed of a dynamic subset of Avalanche
 validators working together to achieve consensus on the state of one or more
-blockchains. Each blockchain is validated by exactly one Subnet, and a Subnet
-can have many blockchains. A validator may be a member of many Subnets.
+blockchains. Each blockchain is validated by exactly one Subnet, while a Subnet
+can host many blockchains.
 
-Avalanche's 3 built-in blockchains: Platform Chain (P-Chain), Contract Chain
-(C-Chain) and Exchange Chain (X-Chain) are validated and secured by all the
-Avalanche validators which compose a special Subnet and is referred as the
-Primary Network.
+Avalanche's [Primary Network](avalanche-platform.md) is a special Subnet running three blockchains:
+
+- The Platform Chain [(P-Chain)](avalanche-platform#platform-chain-p-chain)
+- The Contract Chain [(C-Chain)](avalanche-platform#contract-chain-c-chain)
+- The Exchange Chain [(X-Chain)](avalanche-platform#exchange-chain-x-chain)
 
-By definition, all Subnet validators must also validate the Avalanche Primary Network.
 
 ![image](/img/subnet-validators.png)
 
 (Image adopted from [this article](https://www.coinbase.com/cloud/discover/dev-foundations/intro-to-avalanche-subnets))
 
-Subnets are independent, they specify their own execution logic, determine their
-own fee regime, maintain their own state, facilitate their own networking, and
-provide their own security. They don’t share execution thread, storage[^1] or
-networking with other Subnets including the Primary Network, effectively
-allowing the network to scale up easily while enabling lower latency, higher
-transactions per second (TPS), and lower transaction costs provided by the
-Avalanche Consensus.
+:::info
+Every validator, regardless of Subnet involvement,
+**must** validate the Primary Network.
+:::
 
 ## Advantages
 
-A Subnet manages its own membership, it can create its own token economics and
-rules, and may require that its constituent validators have certain properties.
-
-### Independent Token Economics
-
-Subnets can have their own token economics with their own native tokens and fee
-markets. They can launch their own blockchains with customized virtual machines.
-See [Customize a Subnet](/docs/subnets/customize-a-subnet.md) for more details.
+### Independent Networks
 
-### Compliance
+- Subnets specify their own execution logic, determine their
+own fee regime, maintain their own state, facilitate their own networking, and
+provide their own security. 
+- Each Subnet's performance is isolated from other Subnets in the ecosystem, so increased usage on
+one Subnet won't affect another.
+- Subnets can have their own token economics with their own native tokens, fee
+markets, and incentives determined by the Subnet deployer. 
+- One Subnet can host multiple blockchains with customized [virtual machines](virtual-machines.md).
```

Comment:
Again, I feel like "can validate" is better than "can host" since that's the verbiage used in the past but I'd also be OK with "host."
```suggestion
- One Subnet can host multiple blockchains with customized [virtual machines](virtual-machines.md).
```